README.md

```diff
@@ -32,10 +32,36 @@ In addition to the in-library decoding support, you may use the included `plcras
 `plcrashutil convert --format=iphone example_report.plcrash | symbolicatecrash`
 Future library releases may include built-in re-usable formatters, for outputting alternative formats directly from the phone.
 
-## Usage
+## Integration
 
 PLCrashReporter can be added to your app via [CocoaPods](https://guides.cocoapods.org/using/using-cocoapods.html), [Carthage](https://github.com/Carthage/Carthage#quick-start), [Swift Package Manager](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app), or by manually adding the binaries to your project.
 
+### Integration via Cocoapods
+
+1. Add the following line to your `Podfile`:
+```ruby
+pod 'PLCrashReporter'
+```
+2. Run `pod install` to install your newly defined pod and open the project's `.xcworkspace`.
```

Comment:
```suggestion
1. Run `pod install` to install your newly defined pod and open the project's `.xcworkspace`.
```

Reply:
Add link to cocoapods docs for details

Reply:
The link to cocoapods docs is here: https://github.com/microsoft/plcrashreporter/pull/191/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R37.

Reply:
The suggestion doesn't work actually. It displays "1." for this step if you view the page on github. Should be "2."